lib/mkACI.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+import json
+import os
+from optparse import OptionParser
+from shutil import copyfile
+import subprocess
+import sys
+
+
+def parse_opts():
+    usage = "usage: %prog [options] file.aci manifest custom_env paths..."
+    parser = OptionParser(usage=usage)
+    parser.add_option("--thin", dest="thin")
+    parser.add_option("--dnsquirks", dest="dnsquirks")
+    parser.add_option("--static", dest="static")
+    (options, args) = parser.parse_args()
+    if len(args) < 3:
+        sys.stderr.write("not enough arguments")
+        parser.print_help()
+        sys.exit(1)
+    return options, args
+
+
+def paths_from_graphs(graphs):
+    paths = {}
```

Comment:
Looks like the only benefit would be to maintain the illusion of a true set type, probably a matter of taste in this case.